src/sentry/static/sentry/app/components/organizations/timeRangeSelector/index.jsx

```diff
@@ -52,6 +53,18 @@ const getInternalDate = (date, utc) => {
   }
 };
 
+const DateRangeHook = HookOrDefault({
+  hookName: 'component:date-range',
+  defaultComponent: DateRange,
+});
+
+const SelectorListHook = HookOrDefault({
+  hookName: 'component:selector-list',
+  defaultComponent: ({children}) => {
```

Comment:
I noticed I did this somewhere, can this actually just immediately return children instead of wrapping it in a fragment?